README.md

```diff
@@ -32,32 +32,37 @@ the `solidus_frontend` gem. To make this extension work, you need to
 exclude it and manually include all the other Solidus components.
 
 You need to replace:
-
 ```ruby
 gem 'solidus'
 ```
 
 with:
-
 ```ruby
 gem 'solidus_core'
 gem 'solidus_api'
 gem 'solidus_backend'
 gem 'solidus_sample'
 ```
 
-Install our binary in the system: `gem install solidus_starter_frontend`
+If Solidus was already installed with _solidus_frontend_ you will have to change
+all `Spree::Frontend::Config` in `SolidusStarterFrontend::Config`.
 
-Execute our generator that will copy our component files in your project:
-`solidus_starter_frontend`. Now you can start to customize your local views.
+You have the following 2 install options.
 
-If Solidus was already installed with solidus_frontend you will have to change
-all `Spree::Frontend::Config` in `SolidusStarterFrontend::Config`.
+Run our executable to copy the component files in your project:
+- installing it with: `gem install solidus_starter_frontend`;
+- running in your project root folder: `solidus_starter_frontend`;
+- customizing the local views;
+
+Add our component as gem:
+- adding to your _Gemfile_: `gem 'solidus_starter_frontend'` (**put this line 
+before** `gem 'solidus_auth_devise'` **if you are using it**);
+- overriding the views that needs customizations.
 
 ## Usage
 
-`solidus_starter_frontend` will just install a set of files in your solidus application
-and let you customize them to jump-start your custom store design.
+`solidus_starter_frontend` will just install a set of files in your solidus 
+application and let you customize them to jump-start your custom store design.
```

Comment:
This is true only with option 1. Maybe usage does not make yet sense for this extension?